### DISCUSSION TITLE
Welcome to Fluxor Discussions!

### DISCUSSION THREAD

**Discussion by mrpmorris** [mrpmorris/Fluxor#364] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by mrpmorris** [comment]
The situation hasn't changed.

If you are willing to new-up the feature + effects + reducers yourself and add them to the store then it **_might_** work, but that'd mean you won't get to use `[ReducerMethod]` etc.

I'm not sure store code is really big enough to warrant lazy loading anyway.

**Comment by kleelockerby** [comment]
Peter,
That was not exactly what I wanted to hear 😩

Thank you very much for responding.

Kevin Lockerby
